README.md

```diff
@@ -1,5 +1,10 @@
 # Whitehall
 
+Whitehall is deployed in two modes:
```

Comment:
Pondering if it would be useful to add a sentence explaining what Whitehall is re:
`One or more paragraphs describing the app: what it's used for and how it relates to the rest of GOV.UK. Try to link to existing documentation and other READMEs to help keep the description concise.`

I was thinking it would be useful to know it's classed as a legacy app.

Reply:
> It does open a bit of a can of worms to determine what the legacy status is.

Agreed, the line with _legacy_ is too vague.  

Part of me whats to say something like 
`The oldest GOV.UK app that renders various pages and an internal publishing interface`

However "various" is vague too - is there an ideal intro sentence for Whitehall that would be good use?